### PR TITLE
Adding metrics cell coloring based on threshold

### PR DESCRIPTION
**Pull request by davidjumani**
Screenshot from 2020-08-10 19-23-42

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan package

File: src/components/view/ListView.vue [hunk 1]
```diff
@@ -537,4 +593,16 @@ export default {
     }
 
   }
+
+  .alert-notification-threshold {
+    background-color: rgba(255, 231, 175, 0.75);
+    color: #e87900;
+    padding: 10%;
```
Comment (rhtyd) [inline src/components/view/ListView.vue:600]: @davidjumani instead of percentage should we say 10px?

Comment (davidjumani) [reply src/components/view/ListView.vue:600]: px works fine for a larger screen but percentage works better when with mobile!

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
LGTM

**Pull request merged**